package.json

```diff
@@ -25,11 +25,12 @@
   },
   "homepage": "https://github.com/cronofy/cronofy-node#readme",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "rest": "^1.3.1"
+    "lodash": "^4.17.4",
```

Comment:
As it worked for `3.10.1` would it be better to specify this as:

```js
"lodash": ">=3.0.0 <5.0.0",
```

This would allow the library to be used with a wider range of dependencies. Is there a downside?